brave_paks.gni

```diff
@@ -62,7 +63,8 @@ template("brave_extra_paks") {
       sources += [
         "$root_gen_dir/brave/brave_generated_resources.pak",
         "$root_gen_dir/brave/brave_unscaled_resources.pak",
-        "$root_gen_dir/brave/browser/resources/federated_internals/federated_internals_resources.pak",
+
+        # "$root_gen_dir/brave/browser/resources/federated_internals/federated_internals_resources.pak",
```

Comment:
I don't understand what's going on in this commit, why are we commenting things out instead of removing them? Is there a buildflag that should be attached to these?

Reply:
ts needs changing/updating,however I believe federated learning is not ready to ship yet so best to remove code and add to more recent PRs. cc @ptjames @LorenzoMinto

Reply:
Removed at 18cdeadbf7bde26a53d6ce94a71355b02ed7b0d8.